receiver/hostmetricsreceiver/hostmetrics_receiver_test.go

```diff
@@ -133,7 +133,7 @@ func TestGatherMetrics_EndToEnd(t *testing.T) {
 	cancelFn()
 
 	const tick = 50 * time.Millisecond
-	const waitFor = 10 * time.Second
+	const waitFor = 15 * time.Second
```

Comment:
I had to make this change; else the test wasn't passing locally.

Reply:
Can you please revert for a moment to see what is the issue on CI?

Reply:
reverted.

Reply:
hmm.. we don't see this issue in CI. we can leave it at 10s.